src/app/@ansyn/ansyn/app-effects/effects/core.app.effects.ts

```diff
@@ -80,6 +80,7 @@ export class CoreAppEffects {
 	@Effect()
 	loadOverlays$ = this.actions$
 		.ofType<SetOverlaysCriteriaAction>(CoreActionTypes.SET_OVERLAYS_CRITERIA)
+		.filter(action => action.payload.search !== false)
```

Comment:
add comment // This is a hack, if undefined don't cancel search

Reply:
done